.npmignore

```diff
@@ -0,0 +1,2 @@
+.*
+coverage
```

Comment:
Can you move it to `.gitignore` and remove the `.npmignore`? If there is no `.npmignore`, npm will use the rules from `.gitignore` and the `coverage` folder should not be added to git.

Reply:
`coverage` was already in `.gitignore`, but I do see now that if adding `.nyc_output` to `.gitignore`, it was ok to remove `.npmignore` (confirmed by trying `npm publish --dry-run`). It appears that Git ignores at least directories with a leading `.` whereas npm does not. But indeed the trick was to add `.nyc_output` to `.gitignore` for npm to catch on (without need of `.npmignore`).

Reply:
And updated.